opm/output/data/Wells.hpp

```diff
@@ -544,6 +591,15 @@ namespace Opm {
             buffer.write(this->well_potential_gas);
             buffer.write(this->brine);
             buffer.write(this->alq);
+            //tracer:
+            unsigned int size = this->tracer.size();
+            buffer.write(size);
+            for (const auto& titr : this->tracer) {
```

Comment:
Instead of accessing `.first` and `.second` I would access name and tracer rate with the fancy syntax:
```
for (const auto& [name, rate] : this->tracer) {
      ....
} 
```